helm/reana/values.yaml

```diff
@@ -78,6 +78,16 @@ components:
     file_preview_size_limit: 5242880  # 5 * 1024**2 = 5 MiB
     imagePullPolicy: IfNotPresent
     image: docker.io/reanahub/reana-ui:0.9.1-alpha.4
+    launcher_examples:
+    - name: Roofit
+      url: https://github.com/reanahub/reana-demo-root6-roofit
+      image_url: https://raw.githubusercontent.com/reanahub/reana-demo-root6-roofit/master/docs/plot.png
+    - name: ATLAS RECAST
+      url: https://github.com/reanahub/reana-demo-atlas-recast/
+      image_url: https://raw.githubusercontent.com/reanahub/reana-demo-atlas-recast/master/statanalysis/example_results/limit.png
+    - name: CMS HiggsTauTau
+      url: https://github.com/cms-opendata-analyses/HiggsTauTauNanoAODOutreachAnalysis
+      image_url: https://github.com/cms-opendata-analyses/HiggsTauTauNanoAODOutreachAnalysis/raw/master/plots/npv.png
```

Comment:
Let's remove these from here, so that we use the defaults in reana-ui, which point to the internal images (stored in reana-ui repository) instead of the ones on GitHub?